mscore/musescore.cpp

```diff
@@ -4213,7 +4213,15 @@ void MuseScore::changeState(ScoreState val)
             QAction* a = s->action();
             if (!a)
                   continue;
-            if (enable && (s->key() == "undo"))
+#ifdef SCRIPT_INTERFACE
+            // The states inside the plugin creator are handled internally.
+            // changeState(STATE_PLUGIN_CREATOR) will never be called,
+            // because when the plugin creator gets focus, it sets the state
+            // to STATE_DISABLED.
+            if (s->state() == STATE_PLUGIN_CREATOR)
```

Comment:
Change state should not handle the state of the pluginCreator shortcuts.
It is done in [pluginCreator.cpp -> setState](https://github.com/musescore/MuseScore/pull/5632/files#diff-46d4763e789ee0d1bb3401cbebf030a3R137)